frontend/src/pages/AccountSend.vue

```diff
@@ -414,18 +455,55 @@ function useSendForm() {
     if (!token.value?.address) throw new Error(vm.$i18n.tc('Send.select-a-token'));
     if (!recipientId.value) throw new Error(vm.$i18n.tc('Send.enter-a-recipient'));
 
+    sendMax.value = true;
+
     if (NATIVE_TOKEN.value?.address === token.value?.address) {
       if (!userAddress.value || !provider.value) throw new Error(vm.$i18n.tc('Send.wallet-not-connected'));
       const fromAddress = userAddress.value;
       const recipientAddress = await toAddress(recipientId.value, provider.value);
-      const { ethToSend } = await umbraUtils.getEthSweepGasInfo(fromAddress, recipientAddress, provider.value);
-      humanAmount.value = formatUnits(ethToSend, token.value.decimals);
-      return ethToSend;
+      const { ethToSend } = await umbraUtils.getEthSweepGasInfo(fromAddress, recipientAddress, provider.value, {
+        gasLimit: await estimateNativeSendGasLimit(),
```

Comment:
Can we add a comment about why we need to pass a gas limit in here? I know we discussed it and the reason is slipping my mind right now

Reply:
Haha, if I can just remember myself...

Reply:
Ah, figured it out! e0ceb38